src/ios/AKPlugin.m

```diff
@@ -60,8 +60,19 @@ - (void)loginWithEmail:(CDVInvokedUrlCommand *)command {
 }
 
 - (void)getAccount:(CDVInvokedUrlCommand *)command {
+  if (_accountKit == nil) {
+      _accountKit = [[AKFAccountKit alloc] initWithResponseType:AKFResponseTypeAccessToken];
```

Comment:
I can't test this at the moment (missing the hardware on the holidays). But the thing I don't like is the fact that we are initializing `_accountKit` with access token response. What if user previously logged in with authorization code? Have you experienced the scenario?